topics/teaching/tutorials/assessment/tutorial.md

```diff
@@ -0,0 +1,271 @@
+
+By the end of this session, you will be able to:
+* **Describe** the differences between formative and summative assessment
+* **Tell** why frequent feedback is important
+* **List** and **describe** a few techniques for formative feedback
+
+
+The flowchart below highlights where the assessment and feedback phase falls in the overall Nicholls process. Additional information can be found in {% cite via2020course %} and {% cite nicholls2002developing %}.
+
+![A flow chart of the five phases of the instructional design. The 1st phase is to select the Learning Outcomes, followed by the selection of the corresponding Learning Experiences. The third phase is to select the content that is relevant to the LO. The fourth phase is the development of the necessary assessments for the learners, with the final phase being the evaluation of the effectiveness of the LE and the LO, at which point the flow connects back to the first phase for the refinement of the process.](images/NichollsPhases.png "5 phases of instructional design described by Nicholls et al in Developing teaching and learning in higher education. London, UK: Routledge. pp 51-75")
+
+
+# Assessments
+
+There are different types of assessment and different ways that we can categorise them.
+
+1. **Pre-course assessment** (before the course) - verify the target audience of the course
+2. **Preventive assessment** (beginning of the course) - final adjustments of the course to the reality of the participants
+3. **Formative assessment** (during the course) - pilot in real time if learning is taking place
+4. **Summative assessment** (right after the course) - measure and evaluate the knowledge and skills acquired
+5. **Strategic evaluation** (after the course, long time after the course) - measure the adequacy, quality and impact of the course
+
+![A representation of the mental model for assessment, as a tree. It shows the various categories of assessment, as well as the optimal placement of them during the teaching process.](images/assessment-categories.png "Categorizing assessment")
+
+
+> ### {% icon hands_on %} Diagnostic Assessment - Break-out room discussion  - *{% icon time %} 15 min*
+>
+> - Split into groups of 4, depending on the number of participants.
+> - Discuss how you could collect information from learners at the beginning of a course or lesson and how you can use it.
+> - Write it in a notebook and discuss with the rest of the group.
+{: .hands_on}
+
+
+## Formative vs summative assessment
+
+![A cartoon figure showing the main difference between formative and summative assessment, when making soup. When the chef tastes the soup while preparing, it's formative assessment. When the guests taste the soup after its done, it's summative asssessment.](images/FormativeSummativeAssessment.png "Formative and summative assessment")
```

Comment:
ugh amazing, good alt-text, **THANK YOU** :tada: :yellow_heart: :tada: :yellow_heart: :tada: :yellow_heart: